### PR TITLE
Sync rubocop version

### PR DESCRIPTION
**Pull request by ErikSchierboom**
**Blocked by a bug in Rubocop - awaiting new version**

---

- Sync rubocop version
- Update Rubocop gem
- Format using rubocop

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -25,10 +25,16 @@ jobs:
...
         run: |
-          gem install rubocop -v 1.7.0
-          gem install rubocop-minitest -v 0.11.0
-          gem install rubocop-performance -v 1.10.1
-          gem install rubocop-rails -v 2.9.1
+          # Extract the exact versions of the rubocop gems from Gemfile.lock
+          rubocop_version=$(sed -n -E 's/^    rubocop \((.+)\)/\1/p' Gemfile.lock)
+          rubocop_minitest_version=$(sed -n -E 's/^    rubocop-minitest \((.+)\)/\1/p' Gemfile.lock)
+          rubocop_performance_version=$(sed -n -E 's/^    rubocop-performance \((.+)\)/\1/p' Gemfile.lock)
+          rubocop_rails_version=$(sed -n -E 's/^    rubocop-rails \((.+)\)/\1/p' Gemfile.lock)
```
Comment (ErikSchierboom) [inline .github/workflows/tests.yml:32]: A little bit of sed magic to ensure that we always use the same rubocop gems here as used to format our code

File: Gemfile.lock [hunk 2]
```diff
@@ -339,16 +339,16 @@ GEM
     reverse_markdown (2.0.0)
       nokogiri
     rexml (3.2.5)
-    rubocop (1.18.4)
+    rubocop (1.19.1)
```
Comment (ErikSchierboom) [inline Gemfile.lock]: Upgrading fixed an infinite loop formatting issue

**Review by ErikSchierboom — COMMENTED** [review]
After upgrading, we're now hitting the following bug: https://github.com/rubocop/rubocop/pull/10017

It has been fixed but no new release has yet been built.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (iHiD) [reply .github/workflows/tests.yml:32]: Nice.

**Pull request merged**